### PR TITLE
Drop down that manages blending modes.

### PR DESCRIPTION
**Pull request by rthor**
Missing features:
- Set dropdown to correct scroll position on open.
- Recalculate result on blending mode change.
- Click outside of dropdown to close it.

Demo: http://cl.ly/1d0n2T123t2Z

### CODE REVIEW

**Comment by rthor** [comment]
**Pull request says:** Æ em lónlí! No wan vantz to mörgjs mí...

File: app/components/color-mode.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    Ember.Object.create({name: 'Lighten', isActive: false}),
+    Ember.Object.create({name: 'Color Dodge', isActive: false}),
+    Ember.Object.create({name: 'Color Burn', isActive: false}),
+    Ember.Object.create({name: 'Soft Light', isActive: false}),
+    Ember.Object.create({name: 'Difference', isActive: false}),
+    Ember.Object.create({name: 'Exclusion', isActive: false})
+  ],
+
+  currentMode: Ember.computed('modes.@each.isActive', function() {
+    return this.get('modes').filterBy('isActive').get('firstObject');
```
Comment (eirikurn) [inline app/components/color-mode.js:21]: Interesting. Always learning.

File: app/components/color-mode.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    return `is-${this.get('selectedItemPosition')}-selected`;
+  }),
+
+  actions: {
+    open() {
+      this.set('isOpen', true);
+    },
+
+    switchTo(mode) {
+      this.get('currentMode').set('isActive', false);
```
Comment (eirikurn) [inline app/components/color-mode.js:46]: I wonder if you can `set` a nested property directly somehow.

**Comment by eirikurn** [comment]
Awesome!

Gonna create issues for remaining features

LGTM

File: app/components/color-mode.js (hunk 1 above)
Comment (rthor) [inline app/components/color-mode.js:21]: Yeah, the docs are pretty good !

**Pull request merged**